README.md

```diff
@@ -48,7 +55,11 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+  town[:residents].delete("Belle")
```

Comment:
this will work! maybe consider setting this as a variable ```belle``` and then just pushing that?